frontend/packages/schema/src/parser/sql/postgresql/processSqlInChunks.ts

```diff
@@ -228,8 +240,10 @@ async function processPosition(
   lines: string[],
   startIndex: number,
   chunkSize: number,
+  chunkOffset: number,
   callback: (
     chunk: string,
+    chunkOffset?: number,
```

Comment:
Shouldn't this also be required?
ref: https://github.com/liam-hq/liam/pull/3118#discussion_r2293093507

Reply:
I missed that. Thank you very much.
https://github.com/liam-hq/liam/pull/3118/commits/1f21a131f261a6efcd8d236966f41d22737d819e